alt-tab-macos/logic/Keyboard.swift

```diff
@@ -23,9 +23,15 @@ func listenToGlobalKeyboardEvents(_ delegate: Application) {
                 eventsOfInterest: eventMask,
                 callback: keyboardHandler,
                 userInfo: UnsafeMutableRawPointer(Unmanaged.passUnretained(delegate).toOpaque()))
+        if eventTap == nil {
+            debugPrint("Could not enable keyboard hook; is accessibility enabled?")
+            delegate.keyboardWasHooked(false)
+            return
+        }
```

Comment:
I'm wondering here if libraries like `ShortcutRecorder`, discussed in [this thread](https://github.com/lwouis/alt-tab-macos/issues/17#issuecomment-546788907) would work without the system permission in the first place. I think the old Carbon API don't require that permission.

Ideally we require those permission only because we need to focus the windows, not because we have a global hotkey, I think